examples/check-full.yaml

```diff
@@ -27,8 +27,8 @@ jobs:
           - {os: macos-latest,   r: 'release'}
 
           - {os: windows-latest, r: 'release'}
-          # use 4.1 to check with rtools40's older compiler
-          - {os: windows-latest, r: '4.1'}
+          # use 4.0 or 4.1 to check with rtools40's older compiler
```

Comment:
We could maybe make this comment more generic, like "Also test with oldest support R version on windows, to get full range of Rtools compilers"?

Reply:
I think it is good to have the specific comment, so we know why it is there. Then we'll know when it is time to update or remove it.

Reply:
Yes, when we don't need rtools4 any more, we can remove it. Or we can change it to windows oldre-1,2,3 etc. that will make more sense to test on, if anything.

Reply:
I was thinking we'd continue to test on oldrel-4 because that spans the full range of windows R versions and if the package works on the oldest and the most recent, it likely works on everything in between (especially since we know it does on Linux).

But maybe we should remove this from our default action and only add to packages that have compiled code?